migrations/Version20210416155713.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210416155713 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf('mysql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE matching_context CHANGE xpath xpath LONGTEXT DEFAULT NULL');
```

Comment:
We can `ALTER TABLE matching_context MODIFY xpath LONGTEXT DEFAULT NULL` since we’re not renaming the column

Reply:
Didn't know this one, I'll use it next time :+1: